src/utils/tailwind-util.ts

```diff
@@ -0,0 +1,6 @@
+import { StyleSheet, StyleProp, ViewStyle } from 'react-native';
+import { useTailwind as tailwind } from 'nativewind';
+
+export const useTailwind = (className: string): StyleProp<ViewStyle> => {
```

Comment:
every style won't be ViewStyle, there are more of them.
Also seems like `useTailwind` has been [removed](https://www.nativewind.dev/guides/tailwindcss-react-native#usetailwind)

Reply:
You are right not every style will be ViewStyle, so this has to updated as per the usage since this is the only alternative I've got for using tailwind classes on other styles prop of a component.
> Also seems like useTailwind has been [removed](https://www.nativewind.dev/guides/tailwindcss-react-native#usetailwind)
Somehow it is working as of now

Reply:
> Somehow it is working as of now

This is a *very* loose statement and often becomes hard to debug or cause unexpected behaviour in application. In general, these are pretty poor reasoning. We need to be very thoughtful, and careful about each line of code that we checkin in boilerplate as it has affect across all future applications. Please avoid this in future.